### ISSUE TITLE
第 10 章 Actor-Critic 算法 语法小问题.

### ISSUE THREAD

**Issue by lbgitjp** [boyu-ai/Hands-on-RL#28]
https://hrl.boyuai.com/chapter/2/actor-critic%E7%AE%97%E6%B3%95#102-actor-critic

![pic2](https://user-images.githubusercontent.com/43297325/207500387-b2ecc2cf-c53a-4c71-9135-6626623b3dd3.png)

上图中的"不会产生梯度来更新价值函数"是什么意思?

谢谢!

**Comment by Bowen1911** [comment]
我也有这个疑问，因为之前DQN的代码里面，目标q值后面没有跟`.detach()`
